submissions/NadiaVorontsova/hooli-style-popup/style.css

```diff
@@ -0,0 +1,144 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Arial, sans-serif;
+}
+
+::-webkit-scrollbar {
+    width: 0px; 
+}
+
+.header{
+    padding: 13px 17px;
+    box-shadow: 0px 1px 30px 7px rgb(220, 219, 219);
+}
+
+.header__menu{
+    position: relative;
+}
+
+.header__list{
+    display: flex;
+    justify-content:flex-end;
+    align-items: center;
+}
+
+.header__list li{
```

Comment:
Avoid styling using HTML tags. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid using too specific CSS selectors. [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
thank you for helpful materials.